site/frontend/src/pages/compare/compile/comparisons-table.vue

```diff
@@ -86,6 +86,11 @@ Category: ${metadata.category}
   };
   if (testCase.profile === "opt" && metadata.release_profile !== null) {
     addMetadata(metadata.release_profile);
+  } else if (
+    (testCase.profile === "check" || testCase.profile === "debug") &&
```

Comment:
I'm not sure if we should display these details on `check` builds. Yes, they are treated as `dev` builds, but LTO/debug/CGUs are not (?) relevant for `check` builds, so this can be a bit confusing for whoever displays this data and might think that LTO did affect something for a `check` build.

Reply:
I wasn't sure myself that this was a good move: I thought there may be some metadata we'll add in the future that will affect check builds (e.g. turning on debug assertions, some lints, etc).

It's probably not going to happen any time soon, so we can think about it then. I'll limit this to debug builds for now then.